ClosedXML/Excel/CalcEngine/XLCalcEngine.cs

```diff
@@ -23,6 +23,50 @@ public XLCalcEngine(IXLWorksheet ws) : this(ws.Workbook)
             _ws = ws;
         }
 
+        private IList<IXLRange> _cellRanges;
+
+        /// <summary>
+        /// Get a collection of cell ranges included into the expression. Order is not preserved.
+        /// </summary>
+        /// <param name="expression">Formula to parse.</param>
+        /// <returns>Collection of ranges included into the expression.</returns>
+        public IEnumerable<IXLRange> GetAffectingRanges(string expression)
```

Comment:
Rename to `GetPrecedentRanges`. I'm trying to align with Excel:

![image](https://user-images.githubusercontent.com/145854/38986895-2b895e96-43ce-11e8-9db1-1bb6fe5d3dd2.png)